### ISSUE TITLE
Windows: Accessibility for screen readers

### ISSUE THREAD

**Issue by Oire** [notsecure/uTox#341]
The µTox client for Windows is not accessible to screen reader users.    
As far as I can see, you're using pure Windows API functions, so it's definitely doable as Microsoft provides all needed implementation for keyboard and screen reader accessibility.    
It would be great if you make your client accessible.    
Thanks!

**Comment by GrayHatter** [comment]
stale, please open on new repo if you still care about this
